docs/luigi-client-setup.md

```diff
@@ -25,28 +25,94 @@ Install the client in your project using npm:
 npm install @luigi-project/client
 ```
 
+## Configuration
+
 Import the client in places where you want to use it, depending on the environment of your choice:
+
+### No framework/Svelte
+Add this line to the imports section of the `src/main.js` file:
+```javascript
+import LuigiClient from '@luigi-project/client';
+```
+
+### Angular
+Add this line to the imports section of the `src/app/app.component.ts` file:
+```javascript
+import LuigiClient from '@luigi-project/client';
+```
+
+### SAPUI5/OpenUI5
+Add this to the `webapp/home/Home.controller.js` file: 
+```js 
+sap.ui.define(
+  [
+    'sap/ui/core/mvc/Controller',
+    'sap/ui/core/UIComponent',
+    'luigi/demo/libs/luigi-client/luigi-client'
+  ],
+```
+
+### Vue
+Add this line to the imports section of the `src/main.js` file:
+```js
+import LuigiClient from '@luigi-project/client';
+```
+
+### React
+Add this line to the imports section of the `src/App.js` file:
+
 ```javascript
-var LuigiClient = require('@luigi-project/client');
+import LuigiClient from '@luigi-project/client';
 ```
-or
+### Next.JS
+
+<!-- add-attribute:class:success -->
+> **TIP:** You can find an Next.JS example using Luigi Client [here](core/examples/luigi-example-next/pages/sample1.js).
```

Comment:
`[here](core/examples/luigi-example-next/pages/sample1.js)` This link doesn't seem to work for me when running the website locally. Although it works when opening the Github link. Could you try another link method.